packages/calculator-bigint/etc/calculator-bigint.api.md

```diff
@@ -14,9 +14,7 @@ export const add: BinaryOperation<bigint>;
 // @public (undocumented)
 export const calculator: {
     add: BinaryOperation<bigint, bigint>;
-    compare: BinaryOperation<bigint,
-    ComparisonOperator
-    >;
+    compare: BinaryOperation<bigint, ComparisonOperator>;
```

Comment:
Fixes the reformatting that happened in [e7e9a19](https://github.com/dinerojs/dinero.js/commit/e7e9a19e6eb8e4ff8903867a60c1457a8d241d0c). We'll need to figure out what is going on. I didn't notice the reformatting when I reviewed the PR for this commit. But there were a lot of changes. It should be easier to notice in small PRs, and I will keep a look out for it.